src/js/angular/resources/controllers.js

```diff
@@ -158,25 +55,52 @@ resourcesCtrl.controller('ResourcesCtrl', ['$scope', 'toastr', '$interval', '$ti
                 }
             }
         };
-        $scope.chartMemoryOptions = angular.copy($scope.chartOptions);
-        $scope.chartMemoryOptions.chart.yAxis.tickFormat = function (d) {
-            return d + ' GB';
-        };
-        $scope.chartCPUOptions = angular.copy($scope.chartOptions);
-        $scope.chartCPUOptions.chart.yAxis.tickFormat = function (d) {
-            return d + '%';
-        };
 
-        $scope.garbadgeCollector = function () {
+        $scope.resourceMonitorData.cpuLoad = new CPULoadChartData($translate, angular.copy($scope.chartOptions));
+        $scope.resourceMonitorData.fileDescriptors = new FileDescriptorsChart($translate, angular.copy($scope.chartOptions));
+        $scope.resourceMonitorData.heapMemory = new HeapMemoryChart($translate, angular.copy($scope.chartOptions));
+        $scope.resourceMonitorData.offHeapMemory = new NonHeapMemoryChart($translate, angular.copy($scope.chartOptions));
+        $scope.resourceMonitorData.diskStorage = new DiskStorageChart($translate, angular.copy($scope.chartOptions));
+
+        $scope.firstLoad = true;
+        $scope.activeTab = 'resourceMonitor';
+        $scope.error = '';
+        $scope.loader = true;
```

Comment:
See which of all these variables are really needed to be in the scope, a.k.a. to be accessible through the template and make those which are not needed just some local `private` functions.

Reply:
Done